scripts/dev/ua_fetch.py

```diff
@@ -73,6 +73,7 @@ def filter_list(complete_list, browsers):
             if found:
                 break
     return table
+    # pylint: enable=too-many-nested-blocks
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.